waspc/.gitignore

```diff
@@ -2,6 +2,7 @@
 stack*.yaml.lock
 dist-newstyle
 waspc.cabal
+.vscode
```

Comment:
I am assuming this will get into conflict with another PR where we grouped these and will therefore get taken care of there.